driver/others/memory.c

```diff
@@ -1313,6 +1313,13 @@ void blas_memory_free_nolock(void * map_address) {
   free(map_address);
 }
 
+#ifdef SMP
```

Comment:
I'll note that this `ifdef` is completely superfluous, as this entire section is, I believe, technically under an `#ifdef SMP`.  Nevertheless, it us used in several other places in the surrounding code, so I'm just using it here for consistency's sake.  Later it will be good to clean this up when refactoring this file.